jobs/build/signed-compose/build.groovy

```diff
@@ -102,7 +102,8 @@ def signedComposeStateQE() {
     retry(3) {
         sleep seconds
         seconds = seconds + 30
-        buildlib.elliott("${elliottOpts} change-state --state QE ${advisoryOpt}")
+        out = buildlib.elliott("${elliottOpts} change-state --state QE ${advisoryOpt}")
```

Comment:
Hi @shiywang I guess you need the following to receive what you expect in `out =`
```suggestion
        out = buildlib.elliott("${elliottOpts} change-state --state QE ${advisoryOpt}", [capture: true])
```